src/send_data.py

```diff
@@ -0,0 +1,49 @@
+from ast import alias
+from xmlrpc.client import boolean
+from sensor_temp import getTemp
+from sensor_motion import getMotion 
+from alarm import alarm_on
+from button import button_pressed
+import requests
+import json
+import time
+
+
+URL = 'http://127.0.0.1:8000'
+
+## SEND DATA OUT
+
+valid_data = {}
+
+def create_sensor_data_dict():
+    try:
+        sensor_temp = getTemp()
+        temp_c = round(sensor_temp["temperature_c"], 1)
+        temp_f = round(sensor_temp["temperature_f"], 1)
```

Comment:
Can you floor the floating number like the below implementation?

https://github.com/team-ondo/backend/blob/202e391813b8dfd0af3519a5be2366ee53f01d32/src/utils/common.py#L22-L33

Reply:
Is there are reason for using floor instead of round?

Reply:
There is no particularly strong reason.
Do you want to use round?

Reply:
I think if they are similar we can use round, unless there's a good reason to change.